### DISCUSSION TITLE
General information about spartan/ui

### DISCUSSION THREAD

**Discussion by maku** [spartan-ng/spartan#132] [General]
I would be interested to know more about the idea / background of "spartan/ui".

 I would have put these libraries (radix, shadcn/ui) in the “React” corner. 
 
 What is the general design philosophy behind it (brain / helm)? Is there a documentary about it that you can read? I only found https://github.com/goetzrobin/spartan/blob/main/libs/ui/README.md. 
 
 In general, I find the idea of ​​spartan/ui exciting. It will probably be one of the first UI libs where signals are also used. (and the established ones will probably need a long time to change over)

**Comment by ajitzero** [comment]
This article by Robin is a good intro: https://goetzrobin.github.io/blog/2023-08-24-getting-started-with-spartanui-shadcn-like-ui-components-for-angular

**Reply by maku** [reply]
Thank you…

**Comment by goetzrobin** [comment]
Maybe I should link to that article on the readme! @maku let me know if there’s any other questions you have that the article does not touch on!
